extensions/interactions/GraphInput/directives/graph-input-validation.service.spec.ts

```diff
@@ -132,9 +132,8 @@ describe('GraphInputValidationService', () => {
   it('should expect graph and edit customization arguments', () => {
     expect(() => {
       validatorService.getAllWarnings(
-        // TS ignore is used because we are assigning no customization
-        // args here to test errors.
-        // @ts-ignore
+        // Here we are assigning no customization args here to test errors.
```

Comment:
Drop the second "here".

Also it is still not clear what the error being tested is. I think the ts-expect-error comments need to also specify the error message, similar to ts-ignore.

Reply:
done